### PR TITLE
Node cleanup upgrade

### PR DESCRIPTION
**Pull request by Prannav-k**

### CODE REVIEW

File: app/config/nvalues.py [hunk 1]
```diff
@@ -119,6 +121,7 @@
     MIN_STAKE_AMOUNT = 500000 * pow(10, NEWRL_TOKEN_DECIMAL)
     STAKE_PENALTY_RATIO = 10
     STAKE_CT_ADDRESS = 'ct1111111111111111111111111111111111111115'
+    SNAPSHOT_BLOCKS = 100000
```
Comment (kousthubraja) [inline app/config/nvalues.py:124]: @Prannav-k let's set this to 50k? 500000 blocks

File: app/core/blockchain/updater.py [hunk 2]
```diff
@@ -282,7 +282,7 @@ def global_internal_clock():
     """Reccuring clock for all node level activities"""
     global TIMERS
     global SYNC_STATUS
-
+    logger.info("Global internal clock running")
```
Comment (kousthubraja) [inline app/core/blockchain/updater.py:285]: Make this to debug log. Unnecessary logging on usual nodes.

File: app/core/db/dbmanager.py [hunk 3]
```diff
@@ -37,44 +38,54 @@ def revert_to_last_snapshot():
...
+    snapshot_schedule['snapshot_creation_in_progress'] = False
+    return True
     # Set the next_snapshot schedule variable based on block size
     # Next snapshot increase ~ block size
 
 
 def check_and_create_snapshot_in_thread(block_index):
-    if snapshot_schedule['snapshot_creation_in_progress']:
+    if not check_snapshot(block_index):
+        logger.info("Snapshot creation check is False, will be checked again later")
```
Comment (kousthubraja) [inline app/core/db/dbmanager.py:74]: Make this to debug log. Unnecessary logging on usual nodes.

File: app/core/db/dbmanager.py [hunk 4]
```diff
@@ -37,44 +38,54 @@ def revert_to_last_snapshot():
...
         return False
     # thread = threading.Thread(target=create_block_snapshot,
     #     args = (block_index, ))
     # thread.start()
     # TODO - Check and enable threading
     return create_block_snapshot(block_index)
 
+def check_snapshot(block_index):
+    logger.info("Checking if snapshot can be created now")
```
Comment (kousthubraja) [inline app/core/db/dbmanager.py:83]: Make this to debug log. Unnecessary logging on usual nodes.

**Review by kousthubraja — CHANGES_REQUESTED** [review]

File: app/config/nvalues.py (hunk 1 above)
Comment (Prannav-k) [reply app/config/nvalues.py]: done

**Review by kousthubraja — APPROVED** [review]

**Review by kousthubraja — APPROVED** [review]

**Pull request merged**